README.md

```diff
@@ -1 +1,31 @@
-# hw01
\ No newline at end of file
+---
+title: "Yunshu Hu"
+---
+
+# hw01
+
+## Basic Info
+
+My name is Yunshu Hu, I am currently back home in Jinan, China. 
+
+I am a rising third year undergraduate student majoring in economics.
+
+Here is the most recent picture of me I can find in my phone. It is taken by my best friend.
+
+![A picture of me](Yunshu_pic.jpg)
+
+## Fun Facts
+
+Here are some fun facts about me:
+
+* I love museums. Wherever I travel, I visit the local museums. My favorite museums so far are:
+  * [The Palace Museum](https://en.dpm.org.cn/) in Beijing, China
+  * [The Palace of Versailles](http://en.chateauversailles.fr/) in Versailles, France
+  * And [*The Art Institute of Chicago*](https://www.artic.edu/)! 
+
+* I gained 10 pounds after we started remote learning. Partly because my parents are so good at cooking. 
+
+* The book I am currently reading is **The Great Game: The Emergence of wall Street as a World Power** by John Gordon.
+
+* I miss my UChicago friends soooo much. 
+
```

Comment:
Overall, this is a great intro - there aren't really any things that need improvement. However, your photo was a bit big - I don't think we know a way to make images smaller, but if you personally know a way to make it so it doesn't take up so much of a page, that might be a good thing to do in the future. 

Also, this is just a suggestion - but you could play around with different types of lists (i.e. not just bullets, but numbered lists too) next time! 

Once again, super great intro! :)